### PR TITLE
Update constants.js

### PR DESCRIPTION
**Pull request by SlaveJobs**
- Fix FIDA tokenSymbol.

### CODE REVIEW

File: src/views/Home/Home.tsx [hunk 1]
```diff
@@ -35,8 +35,7 @@ const Home: React.FC = () => {
 			</div>
 			<Spacer size="lg" />
 			<StyledInfo>
-				🏆<b>Pro Tip</b>: BAO-ETH UNI-V2 LP token pool yields 32x more tokens
-				than other pairs.
+				🏆<b>Pro Tip</b>: BAO-ETH UNIV2 LP token pool is weighted at 55000, the biggest of all pairs.
```
Comment (rocky-baoboa) [inline src/views/Home/Home.tsx:38]: Could we change this to:

"55x the base rewards (already factored into displayed APY)"

The weighting at 55000 might be a bit confusing to some.

**Review by rocky-baoboa — CHANGES_REQUESTED** [review]
Alternatively, we could just skip this change in this PR and get the `1INCH`->`FIDA` change in.

File: src/views/Home/Home.tsx (hunk 1 above)
Comment (SlaveJobs) [reply src/views/Home/Home.tsx:38]: Sure, I just think 55x is misleading without a proper explanation.

Comment (rocky-baoboa) [reply src/views/Home/Home.tsx:38]: Maybe we can also link to the pool weights section of the docs as well?

Comment (SlaveJobs) [reply src/views/Home/Home.tsx]: Good, I will open another PR with that!

**Review by rocky-baoboa — APPROVED** [review]

**Pull request merged**